components/gitpod-db/go/workspace_instance_test.go

```diff
@@ -124,29 +124,33 @@ func TestFindRunningWorkspace(t *testing.T) {
 	conn := dbtest.ConnectForTests(t)
 
 	workspace := dbtest.CreateWorkspaces(t, conn, dbtest.NewWorkspace(t, db.Workspace{}))[0]
+	now := time.Now()
+	fiveMinAgo := now.Add(-5 * time.Minute)
+	tenMinAgo := now.Add(-10 * time.Minute)
+	tenDaysAgo := now.Add(-10 * 24 * time.Hour)
```

Comment:
Can you add another entry outside of the 10 day window?

Reply:
you mean a second one?

Reply:
Given the test, it's unclear if this 10 days is exactly on the boundary of included, or exactly on the boundary of excluded (and without injecting the boundary into the function), it can actually randomly fall on either side of the ten day boundary depending on how quickly things execute.

What I'm after that there's definitely a record which falls outside, and definitely a record which falls inside the 10 day period.

Reply:
Ah, makes sense. I'll make it explicit that it is more than 10 days ago.